ReactWindows/ReactNative.Shared/UIManager/UIManagerModule.cs

```diff
@@ -34,17 +43,24 @@ public partial class UIManagerModule : ReactContextNativeModuleBase, ILifecycleE
         /// <param name="reactContext">The React context.</param>
         /// <param name="viewManagers">The view managers.</param>
         /// <param name="uiImplementation">The UI implementation.</param>
+        /// <param name="window">The ApplicationView/Framework Element.</param>
         public UIManagerModule(
             ReactContext reactContext,
             IReadOnlyList<IViewManager> viewManagers,
-            UIImplementation uiImplementation)
+            UIImplementation uiImplementation,
+#if WINDOWS_UWP
+            ApplicationView window)
+#else
+            FrameworkElement window)
+#endif
```

Comment:
I'd prefer not to change the behavior of UWP.  Previously, each time the bounds changed, we grabbed the instance from the APplicationView singleton method, now we cache it, so we could hae failure cases that would occur in different ways if these callbacks somehow end up on the wrong thread.

---
In reply to: [85238878](https://github.com/ReactWindows/react-native-windows/pull/831#discussion_r85238878) [](ancestors = 85238878)